doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -430,178 +234,36 @@ Amazon Sidewalk samples
-
-* :ref:`modem_shell_application` sample:
-
-  * Added ``link modem`` command for initializing and shutting down the modem.
-  * Updated to use the :ref:`at_parser_readme` library instead of the :ref:`at_cmd_parser_readme` library.
-
-* :ref:`nrf_cloud_rest_fota` sample:
-
-  * Added support for setting the FOTA update check interval using the config section in the shadow.
-  * Removed redundant logging now done by the :ref:`lib_nrf_cloud` library.
-
-* :ref:`nrf_cloud_multi_service` sample:
-
-  * Added:
-
-    * The :kconfig:option:`CONFIG_TEST_COUNTER_MULTIPLIER` Kconfig option to multiply the number of test counter messages sent, for testing purposes.
-    * A handler for new nRF Cloud event type ``NRF_CLOUD_EVT_RX_DATA_DISCON`` to stop sensors and location services.
-    * Board support files to enable Wi-Fi scanning for the Thingy:91 X.
-    * The :kconfig:option:`CONFIG_SEND_ONLINE_ALERT` Kconfig option to enable calling the :c:func:`nrf_cloud_alert` function on startup.
-    * Logging of the `reset reason code <nRF9160 RESETREAS_>`_.
-    * The :kconfig:option:`CONFIG_POST_PROVISIONING_INTERVAL_M` Kconfig option to reduce the provisioning connection interval once the device successfully connects.
-
-  * Updated:
-
-    * Wi-Fi overlays from newlibc to picolib.
-    * Handling of JITP association to improve speed and reliability.
-    * Renamed the :file:`overlay_nrf7002ek_wifi_no_lte.conf` overlay to :file:`overlay_nrf700x_wifi_mqtt_no_lte.conf`.
-    * Renamed the :file:`overlay_nrf7002ek_wifi_coap_no_lte.conf` overlay to :file:`overlay_nrf700x_wifi_coap_no_lte.conf`.
-    * The value of the :kconfig:option:`CONFIG_COAP_EXTENDED_OPTIONS_LEN_VALUE` Kconfig option in the :file:`overlay_coap.conf` file.
-      A larger value is required now that the :kconfig:option:`CONFIG_NRF_CLOUD_COAP_DOWNLOADS` Kconfig option is enabled.
-    * Handling of credentials check to disable network if not using the provisioning service.
-
-  * Fixed an issue where the accepted shadow was not marked as received because the config section did not yet exist in the shadow.
-  * Removed redundant logging now done by the :ref:`lib_nrf_cloud` library.
-
-* :ref:`nrf_cloud_rest_device_message` sample:
-
-  * Added:
-
-    * Support for dictionary logs using REST.
-    * The :kconfig:option:`CONFIG_SEND_ONLINE_ALERT` Kconfig option to enable calling the :c:func:`nrf_cloud_alert` function on startup.
-    * Logging of the `reset reason code <nRF9160 RESETREAS_>`_.
-
-  * Updated:
-
-    * Credentials check to also see if AWS root CA cert is likely present.
-    * Credentials check failure to disable network if not using the provisioning service.
-
-  * Removed redundant logging now done by the :ref:`lib_nrf_cloud` library.
-
-* :ref:`nrf_cloud_rest_cell_pos_sample` sample:
-
-  * Removed redundant logging now done by the :ref:`lib_nrf_cloud` library.
-
-* :ref:`smp_svr` sample:
-
-  * Added sysbuild configuration files.
-
+|no_changes_yet_note|
```

Comment:
```
 Wi-Fi
 -----
 
-* The WPA supplicant is now switched to Zephyr upstream's fork instead of |NCS|.
-* Added the :ref:`ug_nrf70_developing_offloaded_raw_tx` page.
+* Added:
+
+   * The :ref:`ug_nrf70_developing_offloaded_raw_tx` page.
+   * Support for :ref:`EAP-TLS <ug_nrf70_wifi_advanced_security_modes>` Enterprise security mode.
+   * Support for :ref:`Platform Security Architecture (PSA) crypto APIs <ug_nrf70_developing_wifi_psa_suppo
rt>` for WPA2™ security profiles.
+
+* Updated:
+
+   * The WPA supplicant is now switched to Zephyr upstream's fork instead of |NCS|.
+   * The WPA supplicant now uses ``kernel heap`` instead of ``application (libc) heap``.
```